lndr-backend/lndr-backend.cabal

```diff
@@ -25,13 +25,15 @@ library
                , Lndr.Db.PushData
                , Lndr.Db.Types
                , Lndr.Db.VerifiedCredits
+               , Lndr.Config
```

Comment:
**Nit:** It looks this list is currently sorted alphabetically, but `Lndr.Config` is being inserted here after `Lndr.Db.`. Should be the first entry in the `other-modules:` list.

Reply:
right!